### ISSUE TITLE
通过Pass增加相应节点，来使能GE Compiler的Super Kernel功能

### ISSUE THREAD

**Issue by LeoLau94** [BD-Seed-HHW/xpu_graph#345]
### Describe the feature🚀

```
import torch
import torch_npu
import torchair as tng
from torchair.configs.compiler_config import CompilerConfig
from torch.fx import symbolic_trace

class Model(torch.nn.Module):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.gamma = torch.empty(1024).npu()

    def forward(self,x):
        with tng.scope.super_kernel('superkernel', ''):
            return torch_npu.npu_rms_norm(x, self.gamma)


model = Model()
x = torch.empty(1024).npu()
config = CompilerConfig()

npu_backend = tng.get_npu_backend(compiler_config=config)

class Backend:
    def __call__(self, graph_module, *args):
        print(graph_module.graph)
        return graph_module



compiled = torch.compile(super, backend=Backend())
compiled(x)
```

通过这个测例我们能发现
```
graph():
    %scope_enter : [num_users=0] = call_function[target=torch.ops.air.scope_enter](args = ([_super_kernel_scope, _super_kernel_options], [superkernel, ]), kwargs = {})                                                                          
    return ()
graph():
    %l_self_gamma : torch.Tensor [num_users=1] = placeholder[target=L_self_gamma]
    %l_x_ : torch.Tensor [num_users=1] = placeholder[target=L_x_]
    %npu_rms_norm : [num_users=2] = call_function[target=torch.ops.npu.npu_rms_norm](args = (%l_x_, %l_self_gamma), kwargs = {})                                                                                                                 
    %getitem : [num_users=1] = call_function[target=operator.getitem](args = (%npu_rms_norm, 0), kwargs = {})
    %getitem_1 : [num_users=1] = call_function[target=operator.getitem](args = (%npu_rms_norm, 1), kwargs = {})
    return (getitem, getitem_1)
graph():
    %scope_exit : [num_users=0] = call_function[target=torch.ops.air.scope_exit](args = (), kwargs = {})
    return ()
```
当前GE的SuperKernel使用方式，主要是通过调用两个函数来实现的scoped SuperKernel

因此我们可以通过在途中增加相应节点来完成这一功能

### References

TBC

### Environment details

PyTorch: 2.5.1
XpuGraph: 0.3.0

**Comment by LeoLau94** [comment]
```
当前sk的限制：
针对A2上，通信算子和TBE算子还不支持
针对AscendC算子是mix1：1的，需要同步支持1：2，这种比较少可以先调起来
moe算子，combine和dispatch在A2上还不支持
```

**Comment by yfchen-byted** [comment]
作为xpu_graph inner compile的后处理 或者说 ge的前处理 直接调用一次superkernel wrapper可能会让控制流更清晰

def wrap_superkernel(gm):

**Comment by wwens7** [comment]
self-own 的 super kernel 也在这个 issue 讨论吗，还是单开一个？有个问题想一块讨论
